vita3k/modules/SceNgsUser/SceNgs.cpp

```diff
@@ -238,7 +238,8 @@ EXPORT(int, sceNgsVoiceLockParams, std::uint32_t voice, std::uint32_t unk1, std:
     buffer_info->data = alloc(host.mem, 10, "SceNgs buffer stub");
     buffer_info->size = 10;
 
-    return STUBBED("Ngs buffer stubbed");
+	return -1;
+    //return STUBBED("Ngs buffer stubbed");
 }
```

Comment:
Ahh is this me? You can forget about it. I was debugging the p4g video when i stumbled across the persona4kindaworks branch and there are some debugging artifacts left.

Reply:
forgot to remove that back yikes